exercises/practice/dnd-character/dnd-character_spec.lua

```diff
@@ -69,121 +69,71 @@ describe('dnd', function()
   end)
 
   describe('ability', function()
-    it('uses 3 largest numbers from scores in descending order', function()
-      assert.equal(9, dnd.ability({ 4, 3, 2, 1 }))
-    end)
-
-    it('uses 3 largest numbers from scores in ascending order', function()
-      assert.equal(9, dnd.ability({ 1, 2, 3, 4 }))
-    end)
-
-    it('uses 3 largest numbers from scores in random order', function()
-      assert.equal(9, dnd.ability({ 2, 4, 3, 1 }))
-    end)
-
-    it('returns 3 with lowest equal numbers', function()
-      assert.equal(3, dnd.ability({ 1, 1, 1, 1 }))
-    end)
-
-    it('returns 18 with highest equal numbers', function()
-      assert.equal(18, dnd.ability({ 6, 6, 6, 6 }))
-    end)
-  end)
-
-  describe('roll_dice', function()
-    it('returns 4 numbers, each in the range 1 to 6', function()
-      for _ = 1, 10 do
-        local scores = dnd.roll_dice()
-        assert.equal(4, #scores)
-        for _, score in ipairs(scores) do
-          assert.equal("integer", math.type(score))
-          assert.lteq(1, score)
-          assert.lteq(score, 6)
-        end
```

Comment:
Now that `ability` takes no arguments, we can simplify and match the canonical data. Additionally, `roll_dice` is no longer needed.